.circleci/config.yml

```diff
@@ -75,6 +91,7 @@ workflows:
   test_and_build:
     jobs:
       - lint
+      - nyan-report
       - unit-test
```

Comment:
Looks like this will run the unit tests twice, right? Do we need that?

Reply:
That commit was made for the purpose of seeing how the output would look on circle ci. We don't need it, I'll revert it.